external/js/cothority/src/network/connection.ts

```diff
@@ -153,7 +153,11 @@ export class WebSocketConnection implements IConnection {
 
         return new Observable((sub) => {
             const url = new URL(this.url.href);
-            url.pathname = `/${this.service}/${message.$type.name.replace(/.*\./, "")}`;
+            let prefix = "";
+            if (url.pathname !== "/") {
+                prefix = url.pathname;
+            }
+            url.pathname =  `${prefix}/${this.service}/${message.$type.name.replace(/.*\./, "")}`;
```

Comment:
Hum, you can use `url.pathname +=` instead.

Reply:
mmm not quite. If there is no pathname, then pathname equals "/", which give `//serviceName...` but its's correct if pathname equals something like "/conode".

Reply:
Ho, you're right, I thought that an URL couldn't contain empty path element